src/redux/books/books.js

```diff
@@ -1,12 +1,12 @@
-import { v4 as uuidv4 } from 'uuid';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const API_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/l5uMLsaRXl0PoGJPW2Vl/books';
 
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
```

Comment:
- [ ] When we add a book then we don't want to refresh the page since you are using AsyncThunk and it is a SinglePage Application :thinking: Please fix this issue :blush: